### PR TITLE
Fix symlink creation

### PR DESCRIPTION
**Pull request by johandahlberg**
Removing `run` as to not confuse folks. That also mean moving the creation of the symlink to a new place. Finally I also decided that it would be better if the symlink was removed and recreated if it exits, as people might change output dirs between runs and in that case it would make sense to take the most recent one.

### CODE REVIEW

File: bcl2fastq/lib/bcl2fastq_utils.py [hunk 1]
```diff
@@ -262,47 +262,29 @@ def construct_command(self):
...
+                             link_target_path))
+            os.symlink(link_target_path, link_path)
         except OSError as e:
             if e.errno == os.errno.EEXIST:
-                log.warning("Symlink from {} to {} already exits, will not recreate it...".
-                            format(self.config.runfolder_input + "/Unaligned",
-                                   self.config.output))
+                log.warning("Symlink from {} to {} already exits, will remove it and recreate it...".
+                            format(link_path, link_target_path))
+                log.warning("Removing link: {}".format(link_path))
```
Comment (withrocks) [inline bcl2fastq/lib/bcl2fastq_utils.py:280]: Perhaps unnecessary, since there is that warning just above it?

**Comment by withrocks** [comment]
Looks cool. Let me know if I can merge it.

File: bcl2fastq/lib/bcl2fastq_utils.py (hunk 1 above)
Comment (johandahlberg) [inline bcl2fastq/lib/bcl2fastq_utils.py:280]: I did it that way because it makes the log easier to read if that ends up on two lines...

**Comment by johandahlberg** [comment]
Go ahead and merge. :+1:

File: bcl2fastq/lib/bcl2fastq_utils.py (hunk 1 above)
Comment (withrocks) [inline bcl2fastq/lib/bcl2fastq_utils.py:280]: Right.

**Pull request merged**